paper-tabs.html

```diff
@@ -285,6 +285,27 @@
           value: 'paper-tab'
         },
 
+        /**
+         * If true, tabs are not automatically selected when focused using the
+         * keyboard.
+         */
+        noAutoselect: {
```

Comment:
Also, talked offline, this is actually a breaking change; this should probably be called `autoselect` and default to false, no?